ports/glutin/window.rs

```diff
@@ -261,11 +261,21 @@ impl Window {
             Event::Resized(width, height) => {
                 self.event_queue.borrow_mut().push(WindowEvent::Resize(Size2D::typed(width, height)));
             }
-            Event::MouseInput(element_state, mouse_button) => {
+            Event::MouseInput(element_state, mouse_button, pos) => {
                 if mouse_button == MouseButton::Left ||
-                                    mouse_button == MouseButton::Right {
-                        let mouse_pos = self.mouse_pos.get();
-                        self.handle_mouse(mouse_button, element_state, mouse_pos.x, mouse_pos.y);
+                    mouse_button == MouseButton::Right {
+                        match pos {
+                            Some(x, y) => {
+                                self.mouse_pos.set(Point2D::new(x, y));
+                                self.handle_mouse(mouse_button, element_state, x, y);
+                                self.event_queue.borrow_mut().push(
+                                WindowEvent::MouseWindowMoveEventClass(Point2D::typed(x as f32, y as f32)));
```

Comment:
Hmm, I'm wrong here. But I do think the solution should be:
1. set the new position
2. push the `MouseWindowMoveEventClass` into `event_queue`
3. call `handle_mouse`

For the move event, should I also call `handle_mouse`? I though `handle_mouse` is only for the click event.